src/ast.hpp

```diff
@@ -1047,21 +1047,25 @@ namespace Sass {
   private:
     ADD_PROPERTY(enum Sass_Separator, separator)
     ADD_PROPERTY(bool, is_arglist)
+    ADD_PROPERTY(enum Sass_List_Delimiter, delimiter)
     ADD_PROPERTY(bool, from_selector)
   public:
     List(ParserState pstate,
-         size_t size = 0, enum Sass_Separator sep = SASS_SPACE, bool argl = false)
+         size_t size = 0, enum Sass_Separator sep = SASS_SPACE, bool argl = false,
+         enum Sass_List_Delimiter delimiter = SASS_NO_DELIMITER)
```

Comment:
Since this changes the function prototype anyway, I would prefer to have the "official" arguments before the "libsass" specific ones (size, sep, del, argl).